developer_guide/installation/system_requirements/system_install_ubuntu_1604.rst

```diff
@@ -143,7 +143,7 @@ PHP 5.6 (supported)
 
     $ sudo apt-get update
     $ sudo apt-get install mongodb
-    $ sudo apt-get install php5-mongo
+    $ sudo apt-get install php-mongo
```

Comment:
Are you sure? I can't find a package of that name in http://packages.ubuntu.com/de/xenial/allpackages
All there is a package named `php-mongodb` and that has a PHP 7.0 dependency:
[http://packages.ubuntu.com/de/xenial/php-mongodb](url)